node/Cargo.toml

```diff
@@ -22,55 +22,58 @@ structopt = '0.3.8'
 hex-literal = '0.3.1'
 
 # local node-specific dependencies
-datahighway-runtime = { version = '3.0.5', path = '../runtime' }
+datahighway-runtime = { version = '3.0.6', path = '../runtime' }
 
 # Substrate dependencies
-frame-benchmarking = '3.1.0'
-frame-benchmarking-cli = '3.0.0'
-module-primitives = { version = '3.0.5', default-features = false, path = '../pallets/primitives' }
-pallet-authority-discovery = '3.0.0'
-pallet-democracy = '3.0.0'
-pallet-im-online = '3.0.0'
-pallet-transaction-payment-rpc = '3.0.0'
-sc-authority-discovery = '0.9.0'
-sc-basic-authorship = '0.9.0'
-sc-cli = { features = ['wasmtime'], version = '0.9.0' }
-sc-client-api = '3.0.0'
-sc-consensus-epochs = { version = '0.9.0' }
-sc-chain-spec = { version = '3.0.0' }
-sc-keystore = { version = '3.0.0' }
-sc-consensus = '0.9.0'
-sc-consensus-aura = '0.9.0'
-sc-consensus-slots = '0.9.0'
-sc-executor = { features = ['wasmtime'], version = '0.9.0' }
-sc-rpc = '3.0.0'
-sc-rpc-api = '0.9.0'
-sc-service = {  version = '0.9.0', default-features = false, features = ['wasmtime'] }
-sc-sync-state-rpc = '0.9.0'
-sc-transaction-pool = '3.0.0'
-sp-api = '3.0.0'
-sp-authority-discovery = '3.0.0'
-sp-block-builder = '3.0.0'
-sp-blockchain = '3.0.0'
-sp-consensus = '0.9.0'
-sp-consensus-aura = '0.9.0'
-sp-core = '3.0.0'
-sc-finality-grandpa = '0.9.0'
-sc-finality-grandpa-rpc = '0.9.0'
-sp-finality-grandpa = '3.0.0'
-sp-inherents = '3.0.0'
-sp-keystore = '0.9.0'
-sc-network = '0.9.0'
-sp-runtime = '3.0.0'
-sp-transaction-pool = '3.0.0'
-sp-consensus-babe = { version = '0.9.0' }
-sc-consensus-babe = { version = '0.9.0' }
-sc-consensus-babe-rpc = { version = '0.9.0' }
-sc-telemetry = { version = '3.0.0' }
-substrate-frame-rpc-system = '3.0.0'
+frame-benchmarking = { git = 'https://github.com/paritytech/substrate', rev = 'a41578d5d27c56f2843d5e513d007e891d0b4129' }
```

Comment:
yes, that will build using dependencies from the latest upstream Substrate commit for when we want additional features or to get bug fixes since we last updated to our fork of the original repository, but if we do a forkless on-chain upgrade using code from the latest Substrate commit but then a hotfix is required to the Substrate repository, then we'll also have to do an urgent forkless on-chain upgrade again to revert the change, when we might not have got any value out of being up to date with the most recent few commits from the upstream that are most susceptible to hotfixes and extra maintenance for us.  i think just make it a habit of watching what changes are being made to the Substrate upstream, sync to the most recent we can with a specific `rev='...'` to make it clear what upstream commit we're up to, and then include that change and test forkless runtime upgrade on our testnet and then on mainnet